Code.gs

```diff
@@ -94,6 +110,18 @@ function getBookInfoWithIsbn(isbn){
 function parseMyList(list) {
 }
 
+//prase list items to comma separated string
```

Comment:
this function is already in the master branch. Let's figure out how to rebase the code next Tuesday.